protocol/streaming/full_node_streaming_manager.go

```diff
@@ -55,8 +56,8 @@ type FullNodeStreamingManagerImpl struct {
 type OrderbookSubscription struct {
 	subscriptionId uint32
 
-	// Initialize the subscription with orderbook snapshots.
-	initialize *sync.Once
+	// Whether the subscription is initialized with snapshot.
+	initialized *atomic.Bool
```

Comment:
Switched to `atomic.Bool` since I need to query it [here](https://github.com/dydxprotocol/v4-chain/pull/2181/files#diff-368bd5a4ee5ee048eba2e17066095318806ae24044b9504c57da5380329e34d4R691)